de.dashup.shared/src/main/java/de/dashup/shared/LayoutModePanel.java

```diff
@@ -0,0 +1,24 @@
+package de.dashup.shared;
+
+public class LayoutModePanel {
```

Comment:
Why are you not using the class `Panel`? It also contains the panel id and size.